### PR TITLE
Switch drivebase and intake motors to using new canivore

### PR DESCRIPTION
**Pull request by jbko6**
high priority PR, please push as soon as possible

### CODE REVIEW

File: src/main/java/frc/team2412/robot/util/Mk4Configuration.java [hunk 1]
```diff
@@ -8,13 +8,25 @@
     private final Mk4SwerveModuleHelper.GearRatio ratio;
     private final int drive, angle, encoder;
     private final double offset;
+    private final String canBus;
 
     public Mk4Configuration(Mk4SwerveModuleHelper.GearRatio rat, int dr, int ang, int enc, double off) {
         ratio = rat;
```
Comment (TAKBS2412) [inline src/main/java/frc/team2412/robot/util/Mk4Configuration.java:14]: Nit: Can reuse the other constructor to make things cleaner.

File: src/main/java/frc/team2412/robot/Hardware.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
     public static final double DRIVETRAIN_FRONT_RIGHT_ENCODER_OFFSET = -Math.toRadians(221.924);
     public static final double DRIVETRAIN_BACK_LEFT_ENCODER_OFFSET = -Math.toRadians(214.980);
     public static final double DRIVETRAIN_BACK_RIGHT_ENCODER_OFFSET = -Math.toRadians(168.398);
+    public static final String DRIVETRAIN_INTAKE_CAN_BUS_NAME = "DrivebaseIntake";
```
Comment (bobtseattle) [inline src/main/java/frc/team2412/robot/Hardware.java:20]: I am wondering if "Canivore" or something else is a better name.  That would cut down on confusion if we end up shifting more to the canivore.

**Review by bobtseattle — COMMENTED** [review]
One possible nit.

**Review by TAKBS2412 — APPROVED** [review]

**Comment by TAKBS2412** [comment]
Should have this merged in by the start of next meeting at the very latest.

**Pull request merged**